src/components/VideoPlayer/index.native.tsx

```diff
@@ -1,23 +1,20 @@
 import React, {forwardRef} from 'react';
 import CONST from '@src/CONST';
 import BaseVideoPlayer from './BaseVideoPlayer';
-import {videoPlayerDefaultProps, videoPlayerPropTypes} from './propTypes';
+import type VideoPlayerProps from './types';
 
-function VideoPlayer({videoControlsStyle, ...props}, ref) {
+function VideoPlayer({videoControlsStyle, ...props}: VideoPlayerProps) {
     return (
         <BaseVideoPlayer
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
             isVideoHovered
             shouldUseSharedVideoElement={false}
             videoControlsStyle={[{bottom: CONST.VIDEO_PLAYER.CONTROLS_POSITION.NATIVE}, videoControlsStyle]}
-            ref={ref}
```

Comment:
Why is this removed? Was it unused? Then why is forwardRef still used in the file? 🤔

Reply:
It's unused. I also removed forwardRef.